src/Rings/mpoly-graded.jl

```diff
@@ -1480,6 +1480,20 @@ function _rational_function_to_power_series(P::QQRelPowerSeriesRing, f)
   return _rational_function_to_power_series(P, numerator(f), denominator(f))
 end
 
+@doc raw"""
+    expand(f::Frac{QQPolyRingElem}, d::Int) -> RelPowerSeries
+
+Given a rational function $f$ over the rationals, expand $f$ as a power series
+with $d$ terms.
```

Comment:
You do not want exactly d terms, but and expansion up to terms of degree d, right?